app/lib/nomis_client/base.rb

```diff
@@ -69,6 +79,41 @@ def update_json_headers(params)
           },
         }.deep_merge(params)
       end
+
+      def paginate_through(path, params = {}, verb = :get)
```

Comment:
maybe we want to keep the same signature as 
`NomisClient::Base.send(verb, path, params) `